components/controller.ts

```diff
@@ -626,6 +626,47 @@ export class Controller {
         return json
     }
 
+    /**
+     * Fixes a contract based on game version.
+     *
+     * An example of this is the location for Holiday Hoarders changing in
+     * HITMAN 3 thus breaking the contract in standalone 2016.
+     *
+     * @param contract The contract to fix.
+     * @param gameVersion The game version.
+     * @returns The fixed contract.
+     * @private
+     */
+    private fixContract(
```

Comment:
Let's add a hook here in case plugins need to run their own transforms

Reply:
Doh! I forgot to do this.